features/ingredients_api/models.py

```diff
@@ -0,0 +1,22 @@
+from sqlalchemy import Integer, String, Float, Boolean
+from sqlalchemy.ext.declarative import declarative_base
+from features import DbBaseModel
+from sqlalchemy.orm import Mapped, mapped_column
+from typing import Optional
+
+Base = declarative_base()
+
+class Ingredient(DbBaseModel):
+
+    __tablename__ = "ingredients"
+
+    id: Mapped[int] = mapped_column(Integer, autoincrement=True, primary_key= True)
+    name: Mapped[str] = mapped_column(String, index=True, nullable=False)
+    category: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
Category needs to be something that is pre defined. Another entity like in recipes or Enum